projects/pgai/pgai/cli.py

```diff
@@ -411,3 +442,15 @@ def cli():
 vectorizer.add_command(vectorizer_worker)
 vectorizer.add_command(download_models)
 cli.add_command(vectorizer)
+
+
+@cli.command()
+@click.option(
+    "-d",
+    "--db-url",
+    type=click.STRING,
+    default="postgres://postgres@localhost:5432/postgres",
+    show_default=True,
+)
+def install(db_url: str) -> None:
```

Comment:
Is `upgrading` the same as `installing` over an existing installation?

Reply:
changes made